I2C_eeprom_cyclic_store.h

```diff
@@ -0,0 +1,239 @@
+#pragma once
+#ifndef CYCLICACCESS_H
```

Comment:
If you use **pragma once** the construction with **ifndef** is not needed

Reply:
I read somewhere that it's adviseable to use both, since some compilers still don't support the **pragma once** directive. Not sure to what extent it is necessary so I'll remove it for consistency with your code.

Reply:
Valid point, 
however most compilers do support #pragma as I did not get any issue in any of my libs about it. 
Furthermore, as your class depends on mine and mine has not both it made less sense.